newIDE/app/src/EventsSheet/InlineParameterEditor.js

```diff
@@ -108,13 +110,31 @@ export default class InlineParameterEditor extends React.Component<
     const instruction = this.props.instruction;
     if (!instruction) return null;
 
+    const onRequestClose = () => {
+      if (this.state.instructionMetadata) {
+        const objectParameterIndex = getObjectParameterIndex(
+          this.state.instructionMetadata
+        );
+        setupInstructionParameters(
+          this.props.globalObjectsContainer,
+          this.props.objectsContainer,
+          instruction,
+          this.state.instructionMetadata,
+          objectParameterIndex
+            ? instruction.getParameter(objectParameterIndex)
```

Comment:
Have you checked on multiple conditions and actions? This will crash if there are no object parameter.